lexers/n/numpy.go

```diff
@@ -0,0 +1,28 @@
+package n
+
+import (
+	"strings"
+
+	. "github.com/alecthomas/chroma" // nolint
+	"github.com/alecthomas/chroma/lexers/internal"
+	"github.com/alecthomas/chroma/pkg/shebang"
+)
+
+// NumPy lexer.
+var NumPy = internal.Register(MustNewLexer(
+	&Config{
+		Name:    "NumPy",
+		Aliases: []string{"numpy"},
+	},
+	Rules{
+		"root": {},
+	},
+).SetAnalyser(func(text string) float32 {
+	matched, _ := shebang.MatchString(text, `pythonw?(3(\.\d)?)?`)
```

Comment:
yeah, this if condition is very long. I like the approach with `matched` variable. Maybe we can even extend it, to fully remove this long line.
```suggestion
	hasPythonShebang, _ := shebang.MatchString(text, `pythonw?(3(\.\d)?)?`)
	...

	if (hasPythonShebang || containsImport) && (hasNumpyImport || hasFromNumpyImport) {
	    return 1.0
	}
```